src/components/Book.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = (props) => {
+  const { title, author } = props;
+  return (
+    <div>
+      <h2>{title}</h2>
+      <h4>{author}</h4>
+    </div>
+
```

Comment:
- [ ] As requested in the projects requirements, there should be a button for removing books in the book component. Kindly add it.